CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to Anomalib
+We welcome your input! 👐
+
+We want to make it as simple and straightforward as possible to contribute to this project, whether it is a:
+
+- Bug Report
+- Discussion
+- Feature Request
+- Creating a Pull Request (PR)
+- Becoming a maintainer
+
+
+## Bug Report
+We use GitHub issues to track the bugs. Report a bug by using our Bug Report Template in [Issues](https://github.com/openvinotoolkit/anomalib/issues/new?assignees=&labels=&template=bug_report.md).
+
+
+## Discussion
+We enabled [GitHub Discussions](https://github.com/openvinotoolkit/anomalib/discussions/) in anomalib to welcome the community to ask questions and/or propose ideas/solutions. This will not only provide a medium to the community to discuss about anomalib but also help us de-clutter [Issues](https://github.com/openvinotoolkit/anomalib/issues/new?assignees=&labels=&template=bug_report.md).
+
+
+## Feature Request
+We utilize GitHub issues to track the feature requests as well. If you are certain regarding the feature you are interested and have a solid proposal, you could then create the feature request by using our [Feature Request Template](https://github.com/openvinotoolkit/anomalib/issues/new?assignees=&labels=&template=feature_request.md) in Issues. If it's still in an idea phase, you could then discuss that with the community in our [Discussion](https://github.com/openvinotoolkit/anomalib/discussions/categories/ideas).
+
+
+## Development & PRs
+We actively welcome your pull requests:
+
+  1. Fork the repo and create your branch from [`development`](https://github.com/openvinotoolkit/anomalib/tree/development).
+  2. If you've added code that should be tested, add tests.
+  3. If you've changed APIs, update the documentation.
+  4. Ensure the test suite passes.
+  5. Make sure your code lints.
+  6. Issue that pull request!
```

Comment:
I think we should have a line which says, "Ensure that you have the right to the code you submit or have obtained it from a source with compatible license."